student.rb

```diff
@@ -0,0 +1,23 @@
+# rubocop: disable Style/OptionalBooleanParameter
+require_relative './person'
+
+class Student < Person
+  def initialize(age, name = 'Unknown', parent_permission = true)
+    super(age, name, parent_permission)
+  end
```

Comment:
- [ ] This constructor extends parent's constructor by adding @classroom and a parameter for it, something like so: 
```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```
Please fix that.